webapp/src/app.tsx

```diff
@@ -1757,14 +1758,14 @@ export class ProjectView
 
     private async loadTutorialTemplateCodeAsync(): Promise<void> {
         const header = pkg.mainEditorPkg().header;
-        if (!header || !header.tutorial || !header.tutorial.templateCode)
+        if (!header || !header.tutorial || !header.tutorial.templateCode || header.tutorial.templateLoaded)
             return;
 
         const template = header.tutorial.templateCode;
 
-        // Delete from the header so that we don't overwrite the user code if the tutorial is
-        // re-opened
-        delete header.tutorial.templateCode;
+        // Mark that the templated has been loaded so that we don't overwrite the
```

Comment:
```suggestion
        // Mark that the template has been loaded so that we don't overwrite the
```
so small feel free to ignore